projects/grandine-zkvm-benchmarking-jimmychu0807.md

```diff
@@ -0,0 +1,72 @@
+# Grandine: zkVMs for Beacon Chain Snarkification
+
+## Motivation
+
+[Description from Grandine team](https://github.com/eth-protocol-fellows/cohort-six/blob/master/projects/project-ideas.md#grandine-zkvms-for-beacon-chain-snarkification).
+
+Grandine team are experimenting with zkVM to enable SNARK-based proving of Beacon's Chain state transition functions.
+
+This project will help advancing on chain snarkification, and eventually zk-proving block in real-time.
+
+## Project description
+
+Grandine team has some success on running SP1 and r0vm with Grandine during block production. Now, we want to run benchmark against other zkVMs, e.g. openVM, Zisk, Pico.
+
+We also want to test out the scalability when running with numerous (100+) validators.
+
+## Roadmap
+
+- Study on Ream implementation of zkVM integration (Jul 27)
+- Study [Ere – Unified zkVM Interface & Toolkit](https://github.com/eth-act/ere) (Jul 27)
+- Check [grandine-zk repo](https://github.com/grandinetech/grandine-zk) (Jul 27)
+- Integrate grandine-zk with [**OpenVM**](https://github.com/openvm-org/openvm), running benchmark for 2-3 validator nodes (Aug 10th)
```

Comment:
What are benchmark metrics? Seems to be the core of the project but it's not defined, benchmark needs to provide clear data for apple to apple comparison